packages/sources/layer2-sequencer-health/src/network.ts

```diff
@@ -41,7 +41,7 @@ export const checkSequencerHealth: NetworkHealthCheck = async (
   network: Networks,
 ): Promise<undefined | boolean> => {
   if (!HEALTH_ENDPOINTS[network]?.endpoint) {
-    Logger.info(`Health endpoint not available for network: ${network}`)
+    Logger.debug(`Health endpoint not available for network: ${network}`)
```

Comment:
Wouldn't it be better to do it for Arbitrum only?

Reply:
I had the same idea but felt wrong hardcoding it for a network. I would have done that if the original log was maybe a warn since it's not problematic to not have the endpoint with fallback checks. Since it didn't seem like something actionable for us or NOPs, I thought I could hide it for all networks behind the debug flag? But I don't mind changing it if you still think that's better.

Reply:
I think it should be actionable, no? If the endpoint is not responding the EA probably won't function as expected, let's discuss with the team just in case

Reply:
So the check is more so that an endpoint is not configured and we have defaults for the other networks except starkware which might be in the same boat as Arbitrum. But you're probably right. I'll drop this in the team channel to check.

Reply:
I think, the log message should be send only in the case of config has been initiated or changed. I may be wrong, but the config isn't changed during the execution. Probably, the is no need to dive into `checkSequencerHealth` if there is nothing to check actually, the method could be filtered out from the [checks list](https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/sources/layer2-sequencer-health/src/endpoint/health.ts#L77) using the config.

Reply:
It makes sense to skip `checkSequencerHealth` when an endpoint isn't configured if we're assuming that when a network doesn't have one it's intentional. I was thinking it'd be better to keep the log for tracing purposes if ever needed but mark it as debug since it's not necessary info. What do you think @qs?

Reply:
If we know that `network` is a closed list, then I agree with @qs. The log ideally should only be triggered at startup or on change (though I don't think EAs can change config dynamically). Whether it's `info` or `debug` wouldn't be as much of an issue then since it wouldn't be triggering constantly. That said, if setting it to `debug` won't cause any unintended consequences, then that's probably the easiest course of action.